flexget/plugins/input/next_trakt_episodes.py

```diff
@@ -75,6 +75,7 @@ def on_task_input(self, task, config):
                     trakt_id = item['show']['ids']['trakt']
                     listed_series[trakt_id] = {
                         'series_name': '%s (%s)' % (item['show']['title'], item['show']['year']),
+                        'series_year': item['show']['year'],
```

Comment:
It's got nothing to do with 'trusted source'. The field `series_year` is not used anywhere, so there's no point starting now. `series` plugin needs a bigger overhaul and that is beyond the scope of this PR.

I can see that it is set here https://github.com/Flexget/Flexget/blob/1b2b13c205a65c39c6f3b0755fc23464db9d64f5/flexget/plugins/list/imdb_list.py#L234, but it shouldn't be.